submission/stats/src/main/kotlin/uk/ac/ebi/stats/model/SubmissionStatType.kt

```diff
@@ -0,0 +1,5 @@
+package uk.ac.ebi.stats.model
+
+enum class SubmissionStatType {
+    NUMBER_VIEWS, UNKNOWN
```

Comment:
lets not allow UNKNOWN as this data is new so all values should be well know

Reply:
Done